web/modules/custom/hoeringsportal_public_meeting/src/Plugin/Block/PublicMeetingSummaryBlock.php

```diff
@@ -73,7 +72,7 @@ public function build() {
     $node = \Drupal::routeMatch()->getParameter('node');
 
     if (!$this->helper->isPublicMeeting($node)) {
-      throw new NotFoundHttpException();
+      return [];
```

Comment:
This should never happen, right, and hence should throw an exception?

Reply:
I believe the build function is for some reason called during indexing of search. Maybe since the block is displayed as a Block field through Display Suite.
So we don't get a node to work with from 
`$node = \Drupal::routeMatch()->getParameter('node');`